micromasters/apps.py

```diff
@@ -0,0 +1,9 @@
+"""
+Django App
+"""
+from django.apps import AppConfig
+
+
+class RootConfig(AppConfig):
+    """AppConfig for this project"""
+    name = 'micromasters'
```

Comment:
Please remove this as we already have a `cms` app which should be used for these changes.